core/storage/src/lib.rs

```diff
@@ -122,6 +124,20 @@ macro_rules! ensure_get {
     }};
 }
 
+macro_rules! impl_get_keys_by_schemas_for {
```

Comment:
these are all the paths mocked we need till now :
```
    clients/07-tendermint-1234567/clientType
    clients/07-tendermint-1234567/clientState
    clients/07-tendermint-1234567/consensusStates/0-1
    connections/connection-1234567
    clients/07-tendermint-1234567/connections
    channelEnds/ports/defaultPort/channels/channel-0
   nextSequenceSend/ports/defaultPort/channels/channel-0
    nextSequenceRecv/ports/defaultPort/channels/channel-0
    nextSequenceAck/ports/defaultPort/channels/channel-0
    commitments/ports/defaultPort/channels/channel-0/sequences/0
    acks/ports/defaultPort/channels/channel-0/sequences/0
    receipts/ports/defaultPort/channels/channel-0/sequences/0
```
it`s looks like some paths use the same prefix . @wenyuanhust @KaoImin